hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -418,8 +468,11 @@ public void onCompactionCompleted(
   public RocksDB getRocksDBInstanceWithCompactionTracking(String dbPath)
       throws RocksDBException {
     final Options opt = new Options().setCreateIfMissing(true)
-        .setCompressionType(CompressionType.NO_COMPRESSION);
-    opt.setMaxBytesForLevelMultiplier(2);
+//        .setWriteBufferSize(1L)  // Default is 64 MB. No idea if this is 1B or 1KB or 1MB. Try it
```

Comment:
looks like it is one byte:  https://github.com/facebook/rocksdb/blob/main/include/rocksdb/options.h#L185-L188